integrated_channels/canvas/client.py

```diff
@@ -117,8 +117,70 @@ def delete_content_metadata(self, serialized_data):
 
     def create_assessment_reporting(self, user_id, payload):
         """
-        Not implemented yet
+        Send assessment level learner data, retrieved by the integrated channels exporter, to Canvas in the form of
+        an assignment and submission.
         """
+        learner_data = json.loads(payload)
+        self._create_session()
+
+        # Retrieve the Canvas user ID from the user's edx email (it is assumed that the learner's Edx
+        # and Canvas emails will match).
+        canvas_user_id = self._search_for_canvas_user_by_email(user_id)
```

Comment:
for clarity: I assume this means the edX `user_id` we're passing in here is actually an email address and not a UUID or numerical id?

Reply:
Yeah, this is an email here. Through each of the specific integrated channels' exporters, we designate what we want the learner data's `remote user id` to be, which should the source of data that links an edx learner to an external learner. In this case, we use email. Part of the reason why the parameter is user_id is because of the `remote_user_id` label being passed to the client. But yeah.. both user ID and email.

Reply:
Ah okay. That answered my would-be follow up question of "why not label it `email`?" so thank you for that great explanation! :D